sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -237,6 +244,9 @@ export interface KeyVaultKeyId {
 }
 
 // @public
+// Missing algos tracked by 
+// https://github.com/Azure/azure-sdk-for-js/issues/11262
+// https://github.com/Azure/azure-sdk-for-js/issues/11261
 export type KeyWrapAlgorithm = "RSA-OAEP" | "RSA-OAEP-256" | "RSA1_5";
```

Comment:
How would it look with the missing algorithms? to add support for them, we can get away with allowing the parameters and sending them through the server, that is the first approach we will be taking (local implementations are not in the budget for this month).